examples/own_dataset/README.md

```diff
@@ -1,4 +1,10 @@
 # Example of using your own dataset
+
+This example shows how to train your own dataset stored in CSV format.
+
+This example is for regression task using `Regressor`.
```

Comment:
I would suggest:

We will solve a regression task with `Regressor`. For a classification setting that makes use of `Classifier`, please refer to the `tox21` example.

Reply:
Please add links to the document of `Classifier` and `Regressor`.

http://chainer-chemistry.readthedocs.io/en/stable/generated/chainer_chemistry.models.Classifier.html#chainer_chemistry.models.Classifier
http://chainer-chemistry.readthedocs.io/en/stable/generated/chainer_chemistry.models.Regressor.html#chainer_chemistry.models.Regressor

Reply:
done